framework/libra-framework/sources/ol_sources/vote_lib/multi_action.move

```diff
@@ -121,33 +149,31 @@ module ol_framework::multi_action {
     expiration_epoch: u64,
   }
 
+  /// Offer struct to manage the proposal and claiming of new authorities.
+  /// - proposed: List of authority addresses proposed
+  /// - claimed: List of authority addresses that have claimed the offer.
+  /// - expiration_epoch: The epoch when each proposed expires.
+  struct Offer has key, store {
+    proposed: vector<address>,
+    claimed: vector<address>,
```

Comment:
I don't think you need the claimed vector here. We will know if it was claimed when the address is added to the governance structs

Reply:
We need the claimed vector for initiated accounts. The account owner can only finalize and cage after the minimum claims occur.